### PR TITLE
cut new branch for multicluster-global-hub.

### PR DESCRIPTION
**Pull request by morvencao**
cut new branch(release-2.8) for multicluster-global-hub.

Signed-off-by: morvencao <lcao@redhat.com>

### CODE REVIEW

File: ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.8.yaml [hunk 1]
```diff
@@ -0,0 +1,188 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+  stolostron_builder_go1.18-linux:
```
Comment (clyang82) [inline ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.8.yaml:6]: Can we add go1.19 image as well? We should upgrade to 1.19 in global-hub side.

Comment (morvencao) [reply ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.8.yaml]: done.

**Pull request merged**